cfg/rippled-example.cfg

```diff
@@ -481,19 +481,19 @@
 #       single host from consuming all inbound slots. If the value is not
 #       present the server will autoconfigure an appropriate limit.
 #
-#   max_unknown_time = [<number> seconds | <number> minutes]
+#   max_unknown_time = <number>
 #
-#       The maximum amount of time an outbound peer connection is allowed to
-#       stay in the "unknown" tracking state. The value can take any value
-#       between 5 and 30 minutes, inclusive. If the value is not present the
-#       server will autoconfigure an appropriate limit.
+#       The maximum amount of time, in seconds, that an outbound connection
+#       is allowed to stay in the "unknown" tracking state. This option can
+#       take any value between 300 and 1800 seconds, inclusive. If the option
+#       is not present the server will autoconfigure an appropriate limit.
```

Comment:
"the server will autoconfigure an appropriate limit"... can I query the server for the currently (auto-)configured limit? Since the automatic one is not documented and might be subject to change, it could be interesting to know. Not 100% necessary though I guess.

Reply:
Not that I can think of.